target/linux/lantiq/base-files/etc/board.d/02_network

```diff
@@ -122,6 +122,8 @@ BTHOMEHUBV2B|BTHOMEHUBV3A)
 	lan_mac=$(mtd_get_mac_ascii uboot_env ethaddr)
 	wan_mac=$(macaddr_add "$lan_mac" 1)
 	ucidef_set_interface_lan 'eth0'
```

Comment:
add the change to your last commit: ```git commit --amend target/linux/lantiq/base-files/etc/board.d/02_network```
force push to your github repo: ```git push --force```

Your PR should update automatically. 

But never do this with a repo where more than one person is working with!